Student.java

```diff
@@ -69,5 +69,35 @@ public void assignSeat(){
         mySeatX=getX();
         mySeatY=getY();
     }
- 
+     
+    public void tableSeven(){
```

Comment:
Doesn't seem to be called anywhere

Reply:
i called it in my student file

Reply:
nevermind i couldn't push my student file